code/API_definitions/network_access_management.yaml

```diff
@@ -0,0 +1,2863 @@
+
+    ### Claims
+    OAuth 2.0 claims are not used by this API to allow for the use of JSON Web Encryption (JWE) access tokens.
+
+    ## General Usage
+    This API is intended for creating, modifying, or deleting isolated networks using the `/isolated-networks` endpoints to build access detail configurations. The `/service-sites` endpoints are used for iterating over a service owner's locations. The exact location may be requested with an additional scope to assist with reconciling information about service sites when the API client operator has an existing relationship with the service owner. Similarly, the `/devices` endpoints are used for iterating over and configuring a service owner's devices.  The hardware address may be requested for the same purpose of reconciling information from another source (e.g., database, persons with physical device access, etc.).
+
+    ## Optional Modes
+
+    ### Default Device
+    When an identity for OAuth 2.0 access token has a "default device", such as a typical residential customer with one service site and one network access device, service site and device scopes MAY be omitted.  Any fields referencing a network access device using the `/isolated-networks` or `/isolated-networks/*` APIs MUST be left blank by API clients.   Network operators SHOULD substitute the missing field with the default device.  API definitions include additional details for error responses so API clients can detect if this mode is available.
+
+    ## Comments for Network Operators
+    The following comments are for Network Operators implementing this API.
+
+    ### Auto-Deleting Isolated Networks
+    When an isolated network is not configured on any device the network configuration and its identifier SHOULD be auto-deleted.  API clients MUST not be expected to delete networks not configured on any device.
+
+    ### Mesh Systems
+    Mesh systems SHOULD be represented as a single device when implementing this API due to the undefined behavior of LAN connectivity across multiple devices with the same isolated network.  The recommendation is to use the gateway hardware address of the mesh system when exposing that field, though the hardware address of any node in the mesh may be used.  If an end-user replaces the equipment, all existing isolated networks SHOULD be synced to the new device representing this mesh.  Alternatively, a synthetic hardware address may also be used if the specification for the address type allows.
+
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+  contact:
+    email: project-manager@cablelabs.com
+
+tags:
+  - name: Service Sites
+  - name: Devices
+  - name: Isolated Networks
+  - name: Reboot Requests
+
+servers:
+  - url: "{apiRoot}/{basePath}"
+    variables:
+      apiRoot:
+        default: http://localhost:5000
+        description: API root
+      basePath:
+        default: cpe-management/v0

```

Comment:
Should it be "network-access-management/v0" instead...? or "nam" or "na-management"...

Reply:
Updated to network-access-management.